### PR TITLE
list user unverified administration

### PR DESCRIPTION
**Pull request by Thomas905**

### CODE REVIEW

File: src/Controller/Admin/UserController.php [hunk 1]
```diff
@@ -69,12 +70,23 @@ public function edit(Request $request, User $user, EntityManagerInterface $entit
...
      */
     public function delete(Request $request, User $user, EntityManagerInterface $entityManager): Response
     {
             $entityManager->remove($user);
             $entityManager->flush();
         return $this->redirectToRoute('admin_user_index');
     }
+
+    /**
+     * @Route("/list/unactivate", name="unactivate")
```
Comment (MorganeL-pro) [inline src/Controller/Admin/UserController.php:83]: Admin index needs to be changed ('Hello admincontroller") in order to add a button and an explanation text of this method.

**Review by MorganeL-pro — DISMISSED** [review]

**Review by RomainClair — CHANGES_REQUESTED** [review]
You should probably regenerate the migration from the new dev

**Review by elodiedaubie — APPROVED** [review]
Migrations files corrected :+1:

**Review by RomainClair — CHANGES_REQUESTED** [review]
The migration file seems invalid

**Review by MorganeL-pro — APPROVED** [review]

**Review by Carolinecrepin — APPROVED** [review]

**Review by RomainClair — APPROVED** [review]

**Pull request merged**